### ISSUE TITLE
Project Proposal

### ISSUE THREAD

**Issue by FenixFireMage** [chasemcintyre105/cmRepository#20]
Project Proposal(5%):
Each project group must submit a single project proposal, which should convey the following information:
• who is in the group
• a description of your desired project
Project groups do not need to match assignment groups. The description may be detailed, but it does not
need to be; “We plan to make a FPS” is a valid description. I would recommend, however, that you attempt
to break the project into component parts and briefly plan how each part might be accomplished.

We should have the Proposal in the README for the Project. Then copy it into the Submit Assignment at https://cilearn.csuci.edu/courses/2876/assignments/39063.

**Comment by chasemcintyre105** [comment]
I agree that uploading the proposal into the README is a good idea. As far as us all collaborating to produce a description we all agree on, and can use/follow as we move forward, I've made a google doc with the assignment prompt that we can all edit. Only people signed into a CSUCI email are able to edit this.
https://docs.google.com/document/d/1NqJCacjt6zChIvsoseH0-icfQ-vIn8mAdVqjtCRxCB8/edit?usp=sharing
Sunday night I will take whatever is in this document and put it onto the README, and push it up to GitHub, so make your contributions/alterations before then!

**Comment by FenixFireMage** [comment]
Submitted file located at https://docs.google.com/document/d/1NrMbQQOe5gffXjM49hb0FsFj6U2TJo6uuJ03lTAK_ys/edit?usp=sharing

**Comment by chasemcintyre105** [comment]
Hey guys, would you all be able to meet up for an hour or two in the next couple days to figure out how we are going to bring together our parts of the project?
We could meet in Broome Library for an hour or two.

Chase's Availability:
Tuesday: available after 5:30 pm
Wednesday: available all day except 8:00-10:00am and 7:00-9:00pm
Thursday: available all day except 3:30-6:00pm

**Comment by davidswepson** [comment]
Definitely, here is my availability
Wednesday: available all day except 6 pm - 9 pm
Thursday: available all day except 4:30 pm - 6 pm
Friday: available all day

**Comment by zeknock** [comment]
Hi. I'm Bina.

I'm all available today.(Thu.)


-----Original Message-----
